### DISCUSSION TITLE
How can I hide who I am sponsoring from profile?

### DISCUSSION THREAD

**Discussion by su-pernova** [community/community#38472] [Other Feature Feedback, Questions, & Ideas] [answered]
Is it possible to hide who I am sponsoring from profile?
I sponsored by 'One time' option, not 'Monthly'.

**Comment by david-engelmann** [comment]
If you go to the sponsors dashboard and navigate to settings, you'll see an option to remove who you sponsor (see screenshot below)!

![image](https://user-images.githubusercontent.com/81196676/200327657-2c48e148-635c-499f-a806-3252f1eefc48.png)

**Reply by JoaccoG** [reply]
Does that option removes the section that shows who you sponsor also?? Because I want to remove that section and I already unpublished my profile but the section is still there

**Comment by alimttth** [comment]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link  this is what it looks like

**Reply by david-engelmann** [reply]
@su-pernova's question was on hiding her sponsors section, not modifying the links in her README!

**Comment by JoaccoG** [comment]
I have the same problem, I want to remove 'Sponsoring' section from my profile (not 'sponsored by' section, note the difference: Sponsoring shows the people you sponsor and Sponsored shows the people that sponsors you), and I already unpublished my profile, yet the 'Sponsoring' section is still there... ¿Did someone found a solution to this?

**Reply by su-pernova** [reply]
@JoaccoG Hi, actually I contacted customer center by creating `Github ticket` and the problem solved.
I think the problem is happening because the sponsoring service is still on demo.

**Comment by kachick** [comment, marked as answer]
I also request this feature. As far as I know, monthly plan can be private but one time plan cannot be hidden with web UI.

**Reply by su-pernova** [reply]
@kachick True. Actually, I solved the problem by contacting customer center by creating `Github ticket`.
I think the problem is happening because the sponsoring service is still on demo.

**Reply by kachick** [reply]
Thank you for sharing with me. I basically prefer offcial features. So will wait to be provided the hide option. 🙏

**Reply by damukas** [reply]
thanks

**Comment by dsdanielpark** [comment]
Has this issue been resolved? Despite being a problem from 2022, there still seems to be no way to remove it.

**Comment by probablynotnmp** [comment]
If you want to keep the sponsors of your GitHub profile confidential you can navigate to the "Sponsorships" section, in your profile settings. Turn on the setting to "Hide sponsored members”, for added privacy.

**Reply by dsdanielpark** [reply]
Here's a very short English version:

I set my to private, but the sponsoring section still shows. I want to remove this section entirely in my github profile.

I thought it worked before, but it's still visible online. Do I need to wait a few days for it to disappear? I wonder why this feature isn't implemented."

**Comment by sarath-dbug** [comment]
If you want to keep the sponsors of your GitHub profile confidential you can navigate to the "Sponsorships" section, in your profile settings. Turn on the setting to "Hide sponsored members”, for added privacy.

**Comment by Dalageo** [comment]
If you prefer to keep the **Sponsoring section** of your profile **private**, follow these steps:

1. Navigate to the Sponsoring tab of your profile, or access it directly using the following link, replacing "your_username" with your actual GitHub's  username: `https://github.com/your_username?tab=sponsoring`
2. In this section, you will find a list of people you have sponsored. Click on the `Manage` button.
3. Select the `Make private` option.

<p align="center">
  <img src="https://github.com/user-attachments/assets/3f592ff4-eab6-46cc-bb1f-1b96b8d531ad" alt="Sponsoring" width="400">
</p>

This will make the Sponsoring section of your profile **private**: it will be hidden from public view, but you will still be able to see it when you view your own profile.
